### PR TITLE
Log pacbio reads flag

### PR DESCRIPTION
**Pull request by sara-javadzadeh**
Enable logging for Pacbio log read data by adding the '--log-pacbio-reads' flag.

### CODE REVIEW

File: advntr/vntr_finder.py [hunk 1]
```diff
@@ -499,7 +502,16 @@ def get_dominant_copy_numbers_from_spanning_reads(self, spanning_reads):
             if logp < rev_logp:
                 vpath = rev_vpath
             observed_copy_numbers.append(get_number_of_repeats_in_vpath(vpath))
-
+            if log_pacbio_reads:
+                repeats = get_number_of_repeats_in_vpath(vpath)
```
Comment (Jong-hun-Park) [inline advntr/vntr_finder.py:506]: We can call `get_number_of_repeats_in_vatph(vpath)` once.

File: advntr/vntr_finder.py [hunk 2]
```diff
@@ -499,7 +502,16 @@ def get_dominant_copy_numbers_from_spanning_reads(self, spanning_reads):
...
+                logging.debug(str(Seq(haplotype)))
+                visited_states = [state.name for idx, state in vpath[1:-1]]
+                if self.read_flanks_repeats_with_confidence(vpath):
```
Comment (Jong-hun-Park) [inline advntr/vntr_finder.py:509]: This requires two arguments; read_sequence is missing.

**Review by Jong-hun-Park — CHANGES_REQUESTED** [review]
One minor comment and one missing argument. Please see the comments.

File: advntr/vntr_finder.py (hunk 2 above)
Comment (sara-javadzadeh) [reply advntr/vntr_finder.py]: Added the read_sequence argument.

File: advntr/vntr_finder.py (hunk 1 above)
Comment (sara-javadzadeh) [reply advntr/vntr_finder.py]: Great point. Fixed it.

**Pull request merged**